### DISCUSSION TITLE
Drawing log error

### DISCUSSION THREAD

**Discussion by aa1135435558** [Chia-Network/chia-blockchain#1821] [General]
Log display error when creating new drawing. Is it because the free space of this drive character is not enough 350G？
![4DD776A8-E8DD-4D0A-B708-CC53C6F5998C](https://user-images.githubusercontent.com/67729044/114347057-fa335580-9b96-11eb-9ab9-064c2d0a3f4c.jpeg)

**Comment by hoffmang9** [comment]
332GiB is 356.5GB

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
